packages/polywrap-client-config-builder/polywrap_client_config_builder/client_config_builder.py

```diff
@@ -0,0 +1,149 @@
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from polywrap_core import Uri, IUriResolver, Env
+from typing import Dict, Any, List, Optional, Union
+
+@dataclass(slots=True, kw_only=True) 
+class ClientConfig():
+    """
+    This Abstract class is used to configure the polywrap client before it executes a call
+    The ClientConfig class is created and modified with the ClientConfigBuilder module
+    """
+    envs: Dict[Uri, Dict[str, Any]]
+    interfaces: Dict[Uri, List[Uri]]
+    resolver: IUriResolver
```

Comment:
We should have a list of this. Also, I would suggest that the methods for this are: `add_resolver` & `add_resolvers`
```suggestion
    resolvers: List[IUriResolver]
```

Reply:
changed this implementation